patches/mesa/0001-Add-Managarm-support.patch

```diff
@@ -47,17 +49,34 @@ index c2843a30766..61e926e9178 100644
  dri_drivers = get_option('dri-drivers')
  if dri_drivers.contains('auto')
 diff --git a/src/gallium/drivers/llvmpipe/lp_texture.c b/src/gallium/drivers/llvmpipe/lp_texture.c
-index 2bf223d66f9..79b1575e0a7 100644
+index 5e8f15d53fa..a260f85b41b 100644
 --- a/src/gallium/drivers/llvmpipe/lp_texture.c
 +++ b/src/gallium/drivers/llvmpipe/lp_texture.c
-@@ -929,7 +929,6 @@ llvmpipe_resource_get_param(struct pipe_screen *screen,
+@@ -962,7 +962,6 @@ llvmpipe_resource_get_param(struct pipe_screen *screen,
     default:
        break;
     }
 -   assert(0);
     *value = 0;
     return false;
  }
+diff --git a/src/util/build_id.c b/src/util/build_id.c
+index 8b4f8f30afb..401eadfea7c 100644
+--- a/src/util/build_id.c
++++ b/src/util/build_id.c
+@@ -21,6 +21,7 @@
+  * IN THE SOFTWARE.
+  */
+ 
++#ifndef __managarm__
+ #ifdef HAVE_DL_ITERATE_PHDR
+ #include <dlfcn.h>
+ #include <link.h>
+@@ -134,3 +135,4 @@ build_id_data(const struct build_id_note *note)
+ }
+ 
+ #endif
++#endif
```

Comment:
Why is this done?

Reply:
Hmm, I don't remember exactly, and a rebuild of mesa without these lines added works too. I'll drop it from the patch.